internal/plugincommon/consensus/threshold.go

```diff
@@ -0,0 +1,47 @@
+package consensus
+
+// Threshold is the minimum number of observations required for a given chain.
```

Comment:
`s/chain/scope/g` and maybe give some concrete examples. Per-chain thresholds is one thing, and I think @asoliman92 is saying token addresses is another thing?